cypress/testcases/settings/settings.spec.ts

```diff
@@ -67,12 +67,12 @@ describe('Set backup target S3', () => {
 
         const backupTarget = Cypress.env('backupTarget');
         settings.setS3BackupTarget({
-            type: 'S3', 
-            endpoint: backupTarget.endpoint, 
-            bucketName: backupTarget.bucketName,
-            bucketRegion: backupTarget.bucketRegion,
-            accessKeyId: backupTarget.accessKey,
-            secretAccessKey: backupTarget.secretKey,
+            type: 'S3',
+            endpoint: "http://172.19.99.205:9000",
+            bucketName: "cypress",
+            bucketRegion: "ecmlab",
+            accessKeyId: "minioadmin",
+            secretAccessKey: "minioadmin",
```

Comment:
This looks like you hardcoded in some stuff instead of grabbing from the config/env file. You should change that.

Reply:
Thanks for the check. 
I have change back the S3 backup target setting. 
Since we are migrating test environment to the new lab machines. I thinks it would be better to create a new PR to handle with S3 backup testcases along with the new minio settings.